demo/demo-springmvc/springmvc-client/src/main/resources/microservice.yaml

```diff
@@ -109,6 +109,20 @@ servicecomb:
     duplicate1: newer
 
 cse:
+  fallback:
```

Comment:
Fallback configuration is never used for any testing.

Reply:
![image](https://user-images.githubusercontent.com/30716999/45295986-1aa85880-b533-11e8-9de3-592ae0c15dd8.png)
it's has been tested in demo